providers/UserProvider.tsx

```diff
@@ -53,6 +53,7 @@ export const UserProvider: FC<Props> = ({ children }) => {
   const [metaData, setMetaData] = useState<any>(null)
   const [similarProfiles, setSimilarProfiles] = useState<any>([])
   const { cre8orNumber, getCre8orNumber } = useCre8orNumber({ address: routerAddress || address })
+  const [dbCre8orNumber, setDBcre8orNumber] = useState("")
```

Comment:
why are we creating a new `dbCre8orNumber` variable?

Shouldn't `cre8orNumber` automatically update when you save the new valua to the DB?